cosmo_tester/test_suites/manager_of_managers/constants.py

```diff
@@ -15,32 +15,63 @@
+                     '{0}/cloudify_spire_plugin-{0}-py27-none-' \
+                     'linux_x86_64-centos-Core.wgn'.format(MOM_PLUGIN_VERSION)  # NOQA
+MOM_PLUGIN_YAML_URL = 'https://github.com/cloudify-cosmo/' \
+                      'cloudify-spire-plugin/releases/download/' \
+                      '{0}/plugin.yaml'.format(MOM_PLUGIN_VERSION)  # NOQA
 
 OS_WGN_FILENAME_TEMPLATE = 'cloudify_openstack_plugin-{0}-py27-none-linux_x86_64-centos-Core.wgn'  # NOQA
-OS_YAML_URL_TEMPLATE = 'http://www.getcloudify.org/spec/openstack-plugin/{0}/plugin.yaml'  # NOQA
+OS_YAML_URL_TEMPLATE = 'https://raw.githubusercontent.com/cloudify-cosmo/cloudify-openstack-plugin/{0}/plugin.yaml'  # NOQA
 OS_WGN_URL_TEMPLATE = 'http://repository.cloudifysource.org/cloudify/wagons/cloudify-openstack-plugin/{0}/{1}'  # NOQA
 
 # This version of the plugin is used by the mom blueprint
-OS_PLUGIN_VERSION = '2.12.0'
+OS_PLUGIN_VERSION = '3.2.2'
 OS_PLUGIN_WGN_FILENAME = OS_WGN_FILENAME_TEMPLATE.format(OS_PLUGIN_VERSION)
 OS_PLUGIN_WGN_URL = OS_WGN_URL_TEMPLATE.format(OS_PLUGIN_VERSION,
                                                OS_PLUGIN_WGN_FILENAME)
 OS_PLUGIN_YAML_URL = OS_YAML_URL_TEMPLATE.format(OS_PLUGIN_VERSION)
 
+UTILITIES_PLUGIN_VERSION = '1.15.0'
+UTIL_PLUGIN_WGN_URL = 'http://repository.cloudifysource.org/cloudify/wagons/' \
+                      'cloudify-utilities-plugin/{0}/' \
+                      'cloudify_utilities_plugin-{0}-py27-' \
+                      'none-linux_x86_64-centos-Core.wgn'.format(
+                          UTILITIES_PLUGIN_VERSION)
+UTIL_PLUGIN_YAML_URL = 'http://www.getcloudify.org/spec/' \
+                       'utilities-plugin/{0}/plugin.yaml'.format(
+                           UTILITIES_PLUGIN_VERSION)
+
+ANSIBLE_PLUGIN_VERSION = '2.4.0'
+ANSIBLE_PLUGIN_WGN_URL = 'http://repository.cloudifysource.org/cloudify/' \
+                         'wagons/cloudify-ansible-plugin/{0}/' \
+                         'cloudify_ansible_plugin-{0}-py27-' \
+                         'none-linux_x86_64-centos-Core.wgn'.format(
+                             ANSIBLE_PLUGIN_VERSION)
+ANSIBLE_PLUGIN_YAML_URL = 'http://www.getcloudify.org/spec/' \
+                          'ansible-plugin/{0}/plugin.yaml'.format(
+                              ANSIBLE_PLUGIN_VERSION)
+
 # The version of the OS plugin used by Hello World Example
-HW_OS_PLUGIN_VERSION = '2.0.1'
+HW_OS_PLUGIN_VERSION = '3.2.2'
 HW_OS_WGN_FILENAME = OS_WGN_FILENAME_TEMPLATE.format(HW_OS_PLUGIN_VERSION)
 HW_OS_PLUGIN_WGN_URL = OS_WGN_URL_TEMPLATE.format(HW_OS_PLUGIN_VERSION,
                                                   HW_OS_WGN_FILENAME)
 HW_OS_PLUGIN_YAML_URL = OS_YAML_URL_TEMPLATE.format(HW_OS_PLUGIN_VERSION)
 
-HELLO_WORLD_URL = 'https://github.com/cloudify-cosmo/cloudify-hello-world-example/archive/4.5.zip'  # NOQA
+HELLO_WORLD_URL = 'https://github.com/cloudify-community/blueprint-examples/releases/download/4.5.5-18/hello-world-example.zip'  # NOQA
```

Comment:
we need to keep the format of <version-number>.zip so we could as change it in bump version.
Do we update the new hello world in bump? I think not